src/remote.sh

```diff
@@ -1,5 +1,6 @@
 # NOTE: It is recommended that src/kw_config_loader.sh be included before this
 # file
+include "$KW_LIB_DIR/kw_config_loader.sh"
```

Comment:
I think it would be good to keep it in order to give some context, its not obvious that  config loader needs to be imported before remote and this makes it explicit